subscriptions/models.py

```diff
@@ -0,0 +1,70 @@
+from adminsortable.models import SortableMixin
+from django.db import models
+from django.db.models import Max
+from parler.models import TranslatableModel, TranslatedFields
+
+
+def get_next_subscription_type_category_order():
+    try:
+        return (
+            SubscriptionTypeCategory.objects.all().aggregate(Max("order"))["order__max"]
+            + 1
+        )
+    except TypeError:
+        return 1
```

Comment:
IMO this seems a bit cryptic at first glance, maybe like

```
order_max = SubscriptionTypeCategory.objects.aggregate(Max("order"))["order__max"]
return order_max + 1 if order_max else 1
```

would be easier to read and a bit more robust (it won't swallow other TypeErrors) ?